### DISCUSSION TITLE
(3024, 'Query execution was interrupted, maximum statement execution time exceeded')

### DISCUSSION THREAD

**Discussion by sunbluers** [hhyo/Archery#2674] [Q&A]
服务超时时间及nginx都调整3600秒，如下：
root     28720     1  0 15:59 ?        00:00:01 /export/data/venv4archery/bin/python /export/data/venv4archery/bin/supervisord -c supervisord.conf
root     28722 28720  0 15:59 ?        00:00:00 /export/data/venv4archery/bin/python /export/data/venv4archery/bin/gunicorn -w 4 -b 127.0.0.1:8888 --timeout 3600 archery.wsgi:application
root     28724 28722  3 15:59 ?        00:03:19 /export/data/venv4archery/bin/python /export/data/venv4archery/bin/gunicorn -w 4 -b 127.0.0.1:8888 --timeout 3600 archery.wsgi:application
root     28725 28722  0 15:59 ?        00:00:28 /export/data/venv4archery/bin/python /export/data/venv4archery/bin/gunicorn -w 4 -b 127.0.0.1:8888 --timeout 3600 archery.wsgi:application
root     28726 28722  3 15:59 ?        00:02:54 /export/data/venv4archery/bin/python /export/data/venv4archery/bin/gunicorn -w 4 -b 127.0.0.1:8888 --timeout 3600 archery.wsgi:application
root     28727 28722  5 15:59 ?        00:05:04 /export/data/venv4archery/bin/python /export/data/venv4archery/bin/gunicorn -w 4 -b 127.0.0.1:8888 --timeout 3600 archery.wsgi:application

但是SQL执行超过10分钟时就会报(3024, 'Query execution was interrupted, maximum statement execution time exceeded')，不知道这个还有哪儿限制？

**Comment by nick2wang** [comment]
MAX_EXECUTION_TIME设置了么？
![image](https://github.com/hhyo/Archery/assets/33473924/2cc40e2b-6245-4905-8539-1443565f5038)

**Comment by sunbluers** [comment]
设置了。
